app/dashboard/dashboard.component.e2e-spec.ts

```diff
@@ -0,0 +1,21 @@
+import { browser, by, element } from 'protractor'
+
+describe('Dashboard', () => {
+
+  beforeEach(() => {
+    browser.get('/dashboard');
+  });
+
+
+  it('should show dashboard', () => {
+    browser.wait(function () {
+      return element(by.css('hip-dashboard md-card h2')).isPresent();
+     },60000);
+
+    expect(element(by.css('hip-dashboard md-card h2')).isPresent()).toEqual(true);
+    expect(element.all(by.css('hip-dashboard md-card h2')).get(0).getText()).toEqual('Dashboard' );
```

Comment:
you refer this element three times. You should create a variable for that to improve readability for your code